README.md

```diff
@@ -1,3 +1,5 @@
 # leDab
 Just ignore this tbh it's not that important.
 Supposedly?!.
+
+*Croissant*
```

Comment:
Is there some kind of Crossiant or another emoji we can add to make it more authentic?